lib.ftd

```diff
@@ -865,15 +868,10 @@ margin-bottom.px: 41
 
 -- end: ftd.column
 
--- ftd.column:
-align-content: center
-align-self: center
-
 -- ftd.image:
 src: $simple-steps-desktop.src
 align-self: center
-
--- end: ftd.column
+margin-top.px: -56
```

Comment:
@priyanka9634 use `background.image` details in doc: https://fastn.io/ftd/common/
Background length: https://fastn.io/ftd/built-in-types/#ftd-background-image

Sample:
```ftd
-- ftd.background-image.length bg-position:
x.px: 0
y.px: -56 

-- ftd.background-image bg-image: 
src: $assets.files.static.carousel.how-it-works.svg
position: $bg-position
repeat: no-repeat
```
apply above to your `ftd.column` or `ftd.row` component:
```ftd
-- -- ftd.row:
width: fill-container
background.image: $bg-image
```

Reply:
@gsalunke referencing issue in background-position. @Heulitig will look into it.

Reply:
@priyanka9634 resolving this considering `ftd` issue.